typings/shared/index.d.ts

```diff
@@ -27,7 +27,4 @@ declare module "*.svg" {
 
 // Hot fix - check https://github.com/DefinitelyTyped/DefinitelyTyped/issues/45927 for a fix.
```

Comment:
Please delete this while section, including the comment and the `declare namespace ...` statement.